README.md

```diff
@@ -8,8 +8,8 @@
 [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](CODE_OF_CONDUCT.md)
 
 > :warning: There are two major versions of UI5 Tooling currently available:
-> - [UI5 Tooling v2](https://sap.github.io/ui5-tooling/stable)- Stable version.
-> - [UI5 Tooling v3](https://sap.github.io/ui5-tooling/next)- Development version.
+> - [UI5 Tooling v2](https://sap.github.io/ui5-tooling/v2)- In maintenance.
```

Comment:
Better to have v3 listed first. Would just simplify to "Stable Version" and "Maintenance Version".

If you feel cheerful today, put a :tada: after v3, and omit the :warning:

Reply:
done. Great idea!